taskt/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs

```diff
@@ -82,22 +84,54 @@ public int DebugLine
                 {
                     try
                     {
-                        
+                        IsScriptRunning = true;
                         _selectedTabScriptActions.EnsureVisible(_debugLine - 1);
                     }
                     catch (Exception)
                     {
                         //log exception?
                     }
                 }
+                else if (_debugLine == 0)
+                {
+                    IsScriptRunning = false;
+                    IsScriptSteppedOver = false;
+                    IsScriptSteppedInto = false;
+                    stepIntoToolStripMenuItem.Visible = false;
+                    stepOverToolStripMenuItem.Visible = false;
+                    pauseToolStripMenuItem.Visible = false;
+                    cancelToolStripMenuItem.Visible = false;
+                    if (uiPaneTabs.TabPages.Count == 3)
+                        uiPaneTabs.TabPages.RemoveAt(2);
```

Comment:
Would you be able to provide a bit of context for what the final tab represents at count == 3?